server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/shared/_go-variables.scss

```diff
@@ -16,9 +16,13 @@
 
 
 
-$primary       : #94399e;
-$secondary     : #40d9da;
-$tertiary      : #F5A623;
+$go-primary       : #94399e;
+$go-secondary     : #40d9da;
+$go-tertiary      : #F5A623;
 $light-grey    : #e2e2e2;
 $grey          : #535353;
 $dark-grey     : #1e1e1e;
+
+$border-color: #cccccc;
+
+$global-border-radius:3px;
```

Comment:
There's already a variables.scss, what's go-variables for, could we merge them together?